package custom;

message another_message {
    repeated int32 num_vals = 1;
}
